fe/fe-core/src/main/java/com/starrocks/scheduler/mv/MVMaintenanceJob.java

```diff
@@ -250,9 +250,12 @@ private void buildPhysicalTopology() throws Exception {
         int taskId = 0;
         for (Map.Entry<PlanFragmentId, CoordinatorPreprocessor.FragmentExecParams> kv : fragmentExecParams.entrySet()) {
             CoordinatorPreprocessor.FragmentExecParams execParams = kv.getValue();
-            List<TExecPlanFragmentParams> tParams =
-                    execParams.toThrift(instanceIds, descTable, dbIds, enablePipeline, tabletSinkDop, tabletSinkDop);
+            Set<TUniqueId> instanceIds =
```

Comment:
These code has been changed in my other PR, I think you could ignore the job execution, otherwise it would be rewritten later